manifests/s/SpaceTime/Sheas/Cealer/1.1.1/SpaceTime.Sheas.Cealer.locale.en.yaml

```diff
@@ -0,0 +1,33 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: SpaceTime.Sheas.Cealer
+PackageVersion: 1.1.1
+PackageLocale: en
+Publisher: Space Time
+PublisherUrl: https://www.spacetimee.xyz
+PrivacyUrl: https://thoughts.teambition.com/share/6264eda98adeb10041b92fda#title=Sheas_Cealer_%E9%9A%90%E7%A7%81%E6%94%BF%E7%AD%96
+Author: Space Time
+PackageName: Sheas Cealer
+PackageUrl: https://github.com/SpaceTimee/Sheas-Cealer
+License: No License
+Copyright: Copyright © 2077
+ShortDescription: Just Ceal It
+Description: Just Ceal It

```

Comment:
Please provide a better description of what this application is, what it does, or what it is used for